src/components/views/messages/MImageBody.tsx

```diff
@@ -446,6 +447,21 @@ export default class MImageBody extends React.Component<IBodyProps, IState> {
             gifLabel = <p className="mx_MImageBody_gifLabel">GIF</p>;
         }
 
+        let banner: JSX.Element;
+        const isTimeline = [
+            TimelineRenderingType.Room,
+            TimelineRenderingType.Search,
+            TimelineRenderingType.Thread,
+            TimelineRenderingType.Notification,
+        ].includes(this.context.timelineRenderingType);
```

Comment:
Is this trying to make sure we don't show the banner in the file panel or something else? Perhaps a comment would help here to be a little more explicit about why are we doing this